privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -764,6 +764,12 @@ def getUserList(self, search_dict=None):
         if self.uidtype.lower() != "dn":
             attributes.append(str(self.uidtype))
 
+        unknown_search_keys = list(filter(lambda x: x not in self.userinfo.keys(), search_dict.keys()))
```

Comment:
`unknown_search_keys = [x for x in search_dict.keys() if x not in self.userinfo.keys()]` ?